README.md

```diff
@@ -36,18 +36,26 @@ There are two src codes within this repository [mesh_generator](./src/mesh_gener
 
 ## Building
 
-A bash script is provided for building ([build.sh](./build.sh)). However, there are some instructions below for building without this script.
+A bash [build.sh](./build.sh) script is provided for building all the source code and tests found in this repository. 
+Optionally, the script can also install the project dependacies.  
+
+The script provides both *CMake* and *fpm* backends for building the project, which can be ran with `./build --build-cmake` or `./build --build-fpm` respectively.  
+Note that building with CMake requires a local installation of pFUnit, and fpm  
```

Comment:
I don't understand this sentence. Does building with `CMake` require a local installation of `fpm`, or is the second part of the sentence missing something?